appeals/web/src/server/appeals/appeal-details/audit/audit.controller.js

```diff
@@ -10,26 +12,45 @@ export const renderAudit = async (request, response) => {
 	const { appealId } = request.params;
 	const appeal = request.currentAppeal;
 	const auditInfo = await getAppealAudit(request.apiClient, appealId);
+	const caseNotes = await getAppealCaseNotes(request.apiClient, appealId);
 
-	if (auditInfo) {
+	if (auditInfo || caseNotes) {
 		const auditTrails = await Promise.all(
 			auditInfo.map(async (audit) => {
 				const details = await mapMessageContent(appeal, audit.details, audit.doc, request.session);
-				const loggedDate = audit.loggedDate;
+				const loggedDate = new Date(audit.loggedDate);
 				return {
-					date: dateISOStringToDisplayDate(loggedDate),
-					time: dateISOStringToDisplayTime24hr(loggedDate),
+					dateTime: loggedDate.getTime(),
+					date: dateISOStringToDisplayDate(audit.loggedDate),
+					time: dateISOStringToDisplayTime24hr(audit.loggedDate),
 					details,
 					user: await mapUser(audit.azureAdUserId, request.session)
 				};
 			})
 		);
 
+		const caseNotesArray = await Promise.all(
+			caseNotes.map(async (note) => {
+				const createdAt = new Date(note.createdAt);
+				return {
+					dateTime: createdAt.getTime(),
+					date: dateISOStringToDisplayDate(note.createdAt),
+					time: dateISOStringToDisplayTime24hr(note.createdAt),
+					details: 'Case note added: <br>' + note.comment,
+					user: await mapUser(note.azureAdUserId, request.session)
+				};
+			})
+		);
+
+		const bothArraysSorted = [...auditTrails, ...caseNotesArray].sort(
```

Comment:
This might be better named something like `sortedCaseNotesAndAuditEntries` (or you could move the expression inline as the value of the `auditTrails` property below)